open-media-match/omm_config.py

```diff
@@ -8,16 +8,16 @@
 from threatexchange.signal_type.md5 import VideoMD5Signal
 
 # Database configuration
-PRODUCTION = False
+PRODUCTION = True
 DBUSER = "media_match"
 DBPASS = "hunter2"
-DBHOST = "localhost"
+DBHOST = "db"
 DBNAME = "media_match"
```

Comment:
Normally, we'd be passing these values in from the environment, right? Should we load them with os.env?

Reply:
Yeah I don't know what you want to do with these configs. The config itself is passed in as an environment variable already and loaded. Are you suggesting to get rid of this config file usage?

Reply:
I don't actually know! I understand that we need to configure the flask setup, and the standard way to do that appears to be a file, like here. I don't know what happens when a field is present in both the env and the file, but I would guess the file would take precedence.

To bridge the two, os.env copies the env to file, then flask uses the file, I would guess.